source/configure/plugins-configuration-settings.rst

```diff
@@ -190,103 +183,6 @@ Plugin settings
 
 ----
 
-Agenda
```

Comment:
General comment, are we submitting this config info to the repos so it's not lost?

Reply:
Yes. All community supported integration documentation that was available via docs.mm.com, including config settings, have been moved to their respective GitHub repository README.